<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mani</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>700</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_video">
     <attribute name="title">
      <string>main</string>
     </attribute>
     <widget class="QGroupBox" name="gbox_videoLED">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>420</y>
        <width>141</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Imagen</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>121</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalVideoLEDLayout">
        <item>
         <widget class="QPushButton" name="button_imagen">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(255, 224, 47);
font: 75 9pt &quot;Segoe Print&quot;;</string>
          </property>
          <property name="text">
           <string>Tomar Imagen</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gbox_videoOptions">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>420</y>
        <width>481</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Input_Text</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>451</width>
         <height>72</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalVideoOptionsLayout">
        <item>
         <widget class="QTextEdit" name="textInput_videoName">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_stopvideo">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(255, 136, 39);
font: 75 9pt &quot;Segoe Print&quot;;</string>
          </property>
          <property name="text">
           <string>EnviarTexto</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gbox_videoCounter">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>420</y>
        <width>261</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Video</string>
      </property>
      <widget class="QWidget" name="formLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>241</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="button_startvideo">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(255, 136, 39);
font: 75 9pt &quot;Segoe Print&quot;;</string>
          </property>
          <property name="text">
           <string>Iniciar Video</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gbox_imagePreview_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>631</width>
        <height>401</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="title">
       <string>Preview Dog</string>
      </property>
      <widget class="QLabel" name="label_imagePreview_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>611</width>
         <height>361</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>default.jpg</pixmap>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_session">
     <attribute name="title">
      <string>NosoyNada</string>
     </attribute>
     <widget class="QGroupBox" name="gbox_userConnect">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>120</y>
        <width>281</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Connection Options</string>
      </property>
      <widget class="QPushButton" name="button_connectSession">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>210</y>
         <width>122</width>
         <height>23</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(57, 159, 255, 220);
font: 75 9pt &quot;Segoe Print&quot;;
</string>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_user">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>68</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Usuario</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_password">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>120</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Contrase√±a</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textInput_user">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textInput_password">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>150</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_userError">
       <property name="geometry">
        <rect>
         <x>47</x>
         <y>240</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(255, 0, 0);
font: 75 italic 12pt &quot;PibotoLt&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_counterVideoTempCon">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>470</y>
        <width>205</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Videos tomados sesion actual:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_counterVideoTempVar">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>280</y>
        <width>233</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_image">
     <attribute name="title">
      <string>YoTampoco</string>
     </attribute>
     <widget class="QGroupBox" name="gbox_imagePreview">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>631</width>
        <height>401</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="title">
       <string>Preview imagen</string>
      </property>
      <widget class="QLabel" name="label_imagePreview">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>611</width>
         <height>361</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>default.jpg</pixmap>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gbox_imageLED">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>0</y>
        <width>341</width>
        <height>411</height>
       </rect>
      </property>
      <property name="title">
       <string>Led Options</string>
      </property>
      <widget class="QLabel" name="label_imageLED">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>311</width>
         <height>351</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>Recursos/nosoyunled.jpg</pixmap>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gbox_imageOptions">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>450</y>
        <width>351</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Image options</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>331</width>
         <height>72</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalImageOptionsLayout">
        <item>
         <widget class="QTextEdit" name="textInput_imageName">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gbox_imageCounter">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>450</y>
        <width>301</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Image Counter </string>
      </property>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_counterImageTempCon">
          <property name="text">
           <string>Fotos tomadas sesion actual:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_counterImageTempVar">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
